rules/promotions/sentinelone_threat_external_alerts.toml

```diff
@@ -0,0 +1,97 @@
+[metadata]
+creation_date = "2025/08/04"
+integration = ["sentinel_one"]
+maturity = "production"
+promotion = true
+min_stack_version = "8.18.0"
+min_stack_comments = "Introduced support for SentinelOne threat promotion"
+updated_date = "2025/08/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert for each SentinelOne threat written to the configured indices. Enabling this rule allows you
+to immediately begin investigating SentinelOne threat alerts in the app.
+"""
+from = "now-2m"
+index = ["logs-sentinel_one.threat-*"]
+interval = "1m"
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 1000
+name = "SentinelOne Threat External Alerts"
```

Comment:
Just a quick double check for this, in the [issue](https://github.com/elastic/ia-trade-team/issues/673), it is says the name should be `SentinelOne Threat alerts`. Might be fine as is but just double checking.

Reply:
I changed it just to be consistent with the other rules. Should not cause an issue.